src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -131,8 +179,7 @@ const TopicFragment = ({
     if (item.separator) {
       return (
         <>
-          {renderRecentSearch(index)}
-          <DiscoveryTitleSeparator key="topic-title-separator" text="Suggested Communities" />
+          <DiscoveryTitleSeparator key="topic-title-separator" text="Communities for you!" />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text "Communities for you!" seems more user-friendly than the previous "Suggested Communities". Just ensure that this change aligns with your application's language style guide.
```diff
- <DiscoveryTitleSeparator key="topic-title-separator" text="Suggested Communities" />
+ <DiscoveryTitleSeparator key="topic-title-separator" text="Communities for you!" />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->